global.json

```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
     "rollForward": "latestFeature",
-    "version": "8.0.100-rc.1.23463.5"
+    "version": "8.0.0-rc.2.23480.2"
```

Comment:
@vishweshbankwar Could you update this to `8.0.100-rc.2.23502.2`?

Reply:
It looks like this `8.0.0-rc.2.23480.2` is latest AspNetCore runtime version and not the latest SDK version:

![image](https://github.com/open-telemetry/opentelemetry-dotnet/assets/66651184/fe7d9a68-96bc-4cf5-8346-3b4dcd2f1ec2)

Reply:
My bad. I copied it from runtime folder instead of SDK. fixed in https://github.com/open-telemetry/opentelemetry-dotnet/pull/5002

Reply:
> It looks like this `8.0.0-rc.2.23480.2` is latest AspNetCore runtime version and not the latest SDK version:
> 
> ![image](https://user-images.githubusercontent.com/66651184/278745739-fe7d9a68-96bc-4cf5-8346-3b4dcd2f1ec2.png)

👍